docs/readthedocs/source/doc/LLM/Overview/install_cpu.md

```diff
@@ -4,8 +4,18 @@
 
 Install IPEX-LLM for CPU supports using pip through:
 
-```bash
-pip install --pre --upgrade ipex-llm[all] # install the latest ipex-llm nightly build with 'all' option
+```eval_rst	
+.. tabs::
+
+   .. tab:: Linux
+
+      .. code-block:: bash
+         pip install --pre --upgrade ipex-llm[all] --extra-index-url https://download.pytorch.org/whl/cpu # install the latest ipex-llm nightly build with 'all' option
+
+   .. tab:: Windows
+
+      .. code-block:: bash
```

Comment:
![image](https://github.com/intel-analytics/ipex-llm/assets/54161268/7224e0fa-58e6-47d3-9141-5ba42aaf5405)

Wrong syntax here: there should be an empty line beteen `.. code-black:: ` and the contents

Besides, maybe `.. code-block:: cmd` is better